### DISCUSSION TITLE
How to disable image optimization to just publish the static files generated as simple static files?

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#20152] [Help]
Hi guys, how can I disable the images optimization to just export and publish the static files in my apache local server?

Thanks so much!

**Comment by wilsonneto-swe** [comment]
In my layout images, like logo and assets, there is no need to use next/images, right?

**Comment by qnxdev** [comment]
Did you find it?

**Comment by wilsonneto-swe** [comment]
The solution: I'm not using next/image in my static projects anymore...
I think there isn't other way.
